src/main/java/net/dv8tion/jda/api/entities/Activity.java

```diff
@@ -84,6 +86,14 @@
     @Nullable
     Timestamps getTimestamps();
 
+    /**
+     * The emoji (or custom emoji) attached to a custom status.
+     *
+     * @return {@link Emoji}
```

Comment:
I would suggest changing this to `{@link Emoji} or null if no emoji/emote was set` since it's nullable.